bindata/bootkube/config/bootstrap-config-overrides.yaml

```diff
@@ -9,6 +9,10 @@ extendedArguments:
   - "/etc/kubernetes/secrets/kubelet-signer.crt"
   cluster-signing-key-file:
   - "/etc/kubernetes/secrets/kubelet-signer.key"
+  secure-port:
```

Comment:
why do we bother to override this to the same value?

Reply:
Just reverted.

Reply:
From what I understood @deads2k we don't need that bit at all.